README.md

```diff
@@ -56,7 +56,7 @@ kubectl apply -f https://github.com/actions-runner-controller/actions-runner-con
 
 **Helm Deployment:**
 
-__**Note: For all configuration options for the Helm chart see the chart's [README](./charts/actions-runner-controller/README.md)
+__**Note: For all configuration options for the Helm chart see the chart's [README](./charts/actions-runner-controller/README.md)**__
```

Comment:
`__` and `**` are alternative syntax of the same thing, and using `**` is according to [this](https://www.markdownguide.org/basic-syntax/#bold) the best practice for compatibility purposes.

```suggestion
**Note: For all configuration options for the Helm chart see the chart's [README](./charts/actions-runner-controller/README.md)**
```